clash-vexriscv-sim/app/VexRiscvChainSimulation.hs

```diff
@@ -100,8 +100,12 @@ main = do
   logFileA <- openFile logPathA WriteMode
   logFileB <- openFile logPathB WriteMode
 
+  jtagBridge <- vexrJtagBridge 7894
+  putStrLn "JTAG bridge ready"
```

Comment:
Would it be a good idea to add the port we assign to the jtag bridge to the print statement?
Perhaps at the end so you can also use https://hackage.haskell.org/package/base-4.21.0.0/docs/Data-List.html#v:isPrefixOf to match on `JTAG Bridge started on port XXXX` with `isPrefixOf "JTAG Bridge started"`

This would allow us to prevent / detect mixups in port numbers

Reply:
Works for me